main/options.c

```diff
@@ -597,7 +597,7 @@ static vString* getHome (void)
 	}
 }
 
-#if defined(_WIN32) && !(defined(__USE_MINGW_ANSI_STDIO) && defined(__MINGW64_VERSION_MAJOR))
+#if defined(_WIN32) || !defined(HAVE_ASPRINTF)
```

Comment:
This line can be now just `#ifndef HAVE_ASPRINTF`.

Reply:
I will take your word for it.